packages/services/f-analytics/src/services/analytics.mapper.js

```diff
@@ -0,0 +1,75 @@
+import jwtDecode from 'jwt-decode';
+import SHA256 from 'crypto-js/sha256';
+import {
+    COUNTRY_INFO,
+    DEFAULT_APP_TYPE,
+    DEFAULT_APP_ID,
+    IDENTITY_PROVIDERS,
+    GRANT_TYPES
+} from '@/constants';
+
+const getCookie = (name, req) => {
+    if (req && req.headers && req.headers.cookie) {
+        const value = `; ${req.headers.cookie}`;
+        const parts = value.split(`; ${name}=`);
+        if (parts.length === 2) {
+            return parts.pop().split(';').shift();
+        }
+    }
+
+    return undefined;
+};
+
+const mapAnonymousUserId = (userData, req) => {
+    const value = getCookie('je-auser', req);
+    if (value) {
+        userData['a-UserId'] = value;
+    }
+};
+
+const mapUserAgent = (platformData, req) => {
+    let userAgentString;
+    if (typeof (window) !== 'undefined' && navigator) {
```

Comment:
what is the reason for the syntax of this check? `typeof (window) !== 'undefined'` 
Would `window && window.navigator` or even `window?.navigator` not be enough? 
I would suggest prefixing `navigator` with `window` everywhere to avoid confusion with possibly other Object `navigator`